source/Core/Drivers/Font.h

```diff
@@ -93,7 +93,7 @@ const uint8_t WarningBlock24[] = {
     0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xC0, 0x30, 0x0C, 0x02, 0xF1, 0xF1, 0xF1, 0x02, 0x0C, 0x30, 0xC0, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,

```

Comment:
Why all the changes to this file?

Reply:
that was clang-format i can revert it if you want

Reply:
I would prefer a revet. It doesn't occur for me here with clang format in the docker container version :/